libs/base/ui/structure/collapsible/collapsible.component.spec.ts

```diff
@@ -121,4 +121,67 @@ describe('CollapsibleComponent', () => {
       expect(element).toContainElement('summary[tabindex="-1"]');
     });
   });
+
+  describe('when a syncKey is provided', () => {
+    vi.spyOn(globalThis.sessionStorage.__proto__, 'getItem');
```

Comment:
And just wondering, maybe we need to create some service that will persist users actions?

Reply:
We do have a storage service, and like to use it here, but we're in a lower level UI component. hence we can't use that service so easily.